components/sup/src/manager.rs

```diff
@@ -206,6 +206,23 @@ impl ShutdownConfig {
     }
 }
 
+/// Represents a service with a spec file that cannot be loaded and therefore the service cannot
+/// run. A common reason to be unable to load the spec file would be unsatisfied binds.
+#[derive(Deserialize, Serialize)]
+pub struct UnrunnableService {
+    pub spec_ident: PackageIdent,
+    pub error:      String,
+}
+
+impl From<UnrunnableService> for habitat_sup_protocol::types::UnrunnableServiceStatus {
+    fn from(other: UnrunnableService) -> Self {
+        let mut proto = habitat_sup_protocol::types::UnrunnableServiceStatus::default();
+        proto.ident = other.spec_ident.into();
+        proto.error = other.error;
+        proto
```

Comment:
I'm not sure why we tend to use the paradigm with our proto conversions, but we're free to use the shorter and more rustic approach:
```rust
        Self { ident: other.spec_ident.into(),
               error: other.error, }
```